### DISCUSSION TITLE
Spring Boot vs Spring 차이

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#26] [Q&A] [answered]
Spring Boot vs Spring 차이

**Comment by tjdud22** [comment, marked as answer]
스프링(Spring)은 프레임워크이며, 스프링 부트(Spring Boot)는 스프링 프레임워크를 기반으로 한 도구입니다. 스프링은 설정 파일을 작성해야 하지만, 스프링 부트는 자동 설정을 제공하여 간편하게 개발할 수 있습니다.
